Gemfile

```diff
@@ -28,3 +28,7 @@ end
 group :xdb do
   gem 'mysql2', '~> 0.3.16'
 end
+
+group :zk do
+  gem 'zk'
```

Comment:
Fair enough. I thought it was still best practice to specify the desired version in a project's Gemfile.